src/datasets.py

```diff
@@ -3,6 +3,7 @@
 from torchvision.transforms import Resize, ToTensor, Compose
```

Comment:
`import torchvision.transforms as T`, and use everywhere `T.Resize, T.ToTensor, T.Compose`, etc.

Reply:
What's the reason of that change? `T` could mean `torch` or something else. Couldn't it be confusing?